### DISCUSSION TITLE
Is there a way to disable landscape rotation on mobile ?

### DISCUSSION THREAD

**Discussion by lbellego** [fyne-io/fyne#3258] [Q&A] [answered]
My app is not for a landscape view. Is it possible to disable the rotation for mobile ? Like we can do in a manifest. It's for Android.

**Comment by Jacalz** [comment, marked as answer]
You can place your own Android manifest file in the root of your project. It will be used instead of the one supplied by Fyne.

This way, you can use a manifest that suits your needs better. See https://github.com/cjh0613/wormhole-gui/blob/mobile/AndroidManifest.xml for an example on what I mean (had to link to one of the forks as the manifest has been removed upstream due to no longer being needed).

If you wish to copy the Fyne manifest and modify it for your needs, you can find the code here: https://github.com/fyne-io/fyne/blob/6e90820ea9ca4df836db5a99384c51073415571e/cmd/fyne/internal/mobile/manifest.go#L60.

**Reply by Jacalz** [reply]
I assumed from your comment that you already knew how to do it in the manifest. If not, I think http://stackoverflow.com/questions/2730855/ddg#2730894 should do the trick.

**Reply by lbellego** [reply]
perfect ! thanks ! I didn't saw this info on the doc.

**Reply by Jacalz** [reply]
You're welcome. I'm not sure it is documented. We might need to update that. Thanks for bringing it up :)
